.circleci/config.yml

```diff
@@ -1,238 +1,50 @@
+      - path-filtering/set-parameters:
+          base-revision: ${BASE_REVISION}
+          mapping: |
+            kolena/classification/.* classification true
+            tests/integration/classification/.* classification true
+            kolena/detection/.* detection true
+            tests/integration/detection/.* detection true
+            kolena/fr/.* fr true
+            tests/integration/fr/.* fr true
+            kolena/workflow/.* generic true
+            tests/integration/generic/.* generic true
+            kolena/_(api|utils)/.* all true
+            kolena/.* all true
+            examples/.* example true
       - run:
-          name: Run << parameters.pytest-group >> integration tests
-          command: |
-            export KOLENA_TOKEN=${KOLENA_TOKEN}
-            export KOLENA_CLIENT_BASE_URL=${KOLENA_CLIENT_BASE_URL}
-            TEST_GROUP="<< parameters.pytest-group >>"
-            if [ "$TEST_GROUP" = "misc" ]; then
-              poetry run pytest -vv --durations=0 --cov=kolena --cov-branch \
-                --ignore=tests/integration/classification \
-                --ignore=tests/integration/detection \
-                --ignore=tests/integration/generic \
-                --ignore=tests/integration/fr \
-                tests/integration
-            else
-              poetry run pytest -vv --durations=0 --cov=kolena --cov-branch --junitxml=test-results/result.xml \
-                tests/integration/$TEST_GROUP
-            fi
-      - when:
-          # Generate coverage only from one python version
-          condition:
-            equal: [ "3.9", << parameters.python-version >> ]
-          steps:
-            - run:
-                name: Coverage
-                command: |
-                  poetry run coverage xml --data-file .coverage
-            - codecov/upload:
-                file: coverage.xml
-            - store_test_results:
-                path: test-results
-
-  example-test:
-    parameters:
-      python-version:
-        type: string
-        default: "3.9"
-      subproject:
-        type: string
-      resource-class:
-        type: string
-    docker:
-      - image: cimg/python:<< parameters.python-version >>
-    resource_class: << parameters.resource-class >>
-    working_directory: ~/project/examples/<< parameters.subproject >>
-    steps:
-      - checkout:
-          path: ~/project
-      - run: |
-          poetry config experimental.new-installer false
-          poetry config installer.max-workers 10
-          poetry install --no-ansi
-      - run:
-          name: Run pre-commit checks
-          command: |
-            poetry run pre-commit run -a
+          name: Print parameters
+          command: cat /tmp/pipeline-parameters.json
       - run:
-          name: Run << parameters.subproject >> (Python << parameters.python-version >>) integration tests
+          name: Generate config
```

Comment:
This section is the black magic to replace integration test `pytest-group` matrix with actual values from base `continue_config.yml`.